plugins/bulk-import/src/components/PreviewFile/PreviewFileSidebar.tsx

```diff
@@ -224,52 +234,116 @@ export const PreviewFileSidebar = ({
   handleSave: (pullRequest: PullRequestPreviewData, _event: any) => void;
   isSubmitting?: boolean;
 }) => {
+  const { setStatus, status } = useFormikContext<AddRepositoriesFormValues>();
   const classes = useDrawerStyles();
   const bulkImportApi = useApi(bulkImportApiRef);
+  const identityApi = useApi(identityApiRef);
+  const configApi = useApi(configApiRef);
   const [pullRequest, setPullRequest] = React.useState<PullRequestPreviewData>(
     {},
   );
   const [isInitialized, setIsInitialized] = React.useState(false);
 
+  const fetchPullRequestData = async (
```

Comment:
can we abstract this to util and add tests for it

Reply:
This util has an apiRef which is why I didn't move it from this file